app/controllers/api/picks_controller.rb

```diff
@@ -5,7 +5,7 @@ class Api::PicksController < Api::BaseController
   represent_with Api::PickRepresenter
 
   def resources
-    @picks = Pick.named_playlists.page(params[:page])
+    @picks = Pick.named_playlists
```

Comment:
I don't think we want every pick ever to come through the API, am I misunderstanding this?